### PR TITLE
Add pull filter to vpn-seed config

### PR DESCRIPTION
**Pull request by DockToFuture**
**What this PR does / why we need it**:
This pull request adds pull filter to the vpn-seed config.
Depends on https://github.com/gardener/gardener/pull/1287
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```noteworthy operator
Add pull-filter to vpn-seed
```

### CODE REVIEW

File: seed/network-connection.sh [hunk 1]
```diff
@@ -90,6 +90,69 @@ function configure_tcp() {
...
+# calculate netmask for given CIDR (required by openvpn)
+#
+CIDR2Netmask() {
+    local cidr="$1"
+
+    local ip=$(echo $cidr | cut -f1 -d/)
+    local numon=$(echo $cidr | cut -f2 -d/)
+
+    local numoff=$(( 32 - $numon ))
+    local start=
```
Comment (zanetworker) [inline seed/network-connection.sh]: do you need the =?

File: seed/network-connection.sh [hunk 2]
```diff
@@ -90,6 +90,69 @@ function configure_tcp() {
...
+    done
+    while [ "$numoff" -ne "0" ]; do
+        end=0${end}
+        numoff=$(( $numoff - 1 ))
+    done
+    local bitstring=$start$end
+
+    bitmask=$(echo "obase=16 ; $(( 2#$bitstring )) " | bc | sed 's/.\{2\}/& /g')
+
+    local str=
```
Comment (zanetworker) [inline seed/network-connection.sh]: do you need the =?

File: seed/network-connection.sh [hunk 3]
```diff
@@ -90,6 +90,69 @@ function configure_tcp() {
 
 configure_tcp
 
+# for each cidr config, it looks first at its env var, then a local file (which may be a volume mount), then the default
+baseConfigDir="/init-config"
+fileServiceNetwork=
+filePodNetwork=
+fileNodeNetwork=
+[ -e "${baseConfigDir}/serviceNetwork" ] && fileServiceNetwork=$(cat ${baseConfigDir}/serviceNetwork)
```
Comment (zanetworker) [inline seed/network-connection.sh:98]: maybe use`[[` instead of `[`

Comment (DockToFuture) [reply seed/network-connection.sh:98]: don't know if we really need the bash extension here

Comment (DockToFuture) [reply seed/network-connection.sh:98]: the function is the same, it would just save the quotes

**Review by zanetworker — APPROVED** [review]
/lgtm

**Pull request merged**